src/runner.js

```diff
@@ -169,8 +186,13 @@ class Runner {
         return;
       }
 
+      const formatInstance = new MessageFormat(this.options.formaterOptions.language);
```

Comment:
does this work if we declare `const formatInstance = new MessageFormat(this.options.formaterOptions.language)` only once - on line 28 and remove this and ln 132

Reply:
We can add a new property attached to the runner class then ?

Reply:
you could do
this.messageFormat = new MessageFormat(this.options.formaterOptions.language)